src/gui/folder.cpp

```diff
@@ -664,39 +664,40 @@ int Folder::slotWipeErrorBlacklist()
     return _journal.wipeErrorBlacklist();
 }
 
-void Folder::slotWatchedPathChanged(const QString &path, ChangeReason reason)
+void Folder::slotWatchedPathChanged(const QSet<QString> &paths, ChangeReason reason)
```

Comment:
What's the point of passing a set of paths and handling it in a for loop instead of calling this method in a loop?

Note that the name has become outdated. It should be plural now.

Reply:
Right, it's even called `slot...`. What I meant is `slotWatchedPathChanged` should become `slotWatchedPathsChanged`.